README.md

```diff
@@ -1,12 +1,15 @@
-# Connect-4
+# Connect 4
 
-### by Leo Chan, (c)1997
+**Connect 4, otherwise sometimes known as "Four in a Row", is a classic game usually played within a plastic frame, with the objective of getting 4 pieces of the same color adjacent to each other in any direction. This program is a recreation of the game in BeOS/Haiku software.**
 
-![screenshot](screenshot.png)
+![game in real life](irl.jpg)
+
+*This is what Connect 4 looks like in real life. Photo credit: https://www.flickr.com/photos/unloveable/2396019222*
 
+![screenshot](screenshot.png)
 
-This is the game of Connect-4.  The game is played by players taking turns dropping pieces down a column.  The first player to get four consecutive pieces in a row, column, or diagonal wins the game.
+*This is what the game looks like in software.*
 
-Have fun!
+By Leo Chan, (c)1997
 
-Leo 
+*This software is designed to run on BeOS/Haiku operating system. Compatiiblity and stability are not guaranteed with any platform.*
```

Comment:
I don't think these sentences offer vital information. I suggest removing them and putting "by Leo Chan, ©1997" in italics right under the top heading.

Reply:
i s this ok?

Reply:
Better? Sorry, my Markdown skills are still in the works ;P

Reply:
Hmm.. Can't see a difference. it's still a `###` heading...

Reply:
https://thanos-taxi.thanos.taxi/i/qpqn.png it did change.

Reply:
oh darn i was editing the wrong branch

Reply:
solved